python/tvm/target/target.py

```diff
@@ -234,7 +234,10 @@ def micro(model="unknown", options=None):
     trans_table = {
         "host": [],
         "stm32f746xx": ["-mcpu=cortex-m7", "-march=armv7e-m"],
+        "nrf5340dk": ["-keys=arm_cpu", "-mcpu=cortex-m33"],
```

Comment:
Nit: I'm curious as to why the STM has a `-march=...` field while the NRF has a `keys=...` target field. Could we keep it overall consistent? Thoughts on this @areusch

Reply:
TBH, I have no idea. @areusch ?

Reply:
I think this was because we originally intended to enable ARM schedules based on `-march`. i think this was because i had read some documentation that `-march` was the proper way to specify ISA, but it turns out that this is just specific to x86 targets. For ARM targets, it seems `-mcpu` is actually canonical. So, we may need to improve our ISA parser to handle both.

Actually I think `-keys` here is also unnecessary--my apologies. @mdw-octoml can you remove that? I believe it should get auto-added.

Reply:
@areusch Done - PTAL